File: TheSpanishInquisition/app/src/main/java/com/example/konra/thespanishinquisition/LayoutActivity.java
package com.example.konra.thespanishinquisition;

import android.app.ListActivity;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.Gravity;
import android.view.View;
import android.widget.ListView;
import android.widget.Toast;

import com.codemonkeylabs.fpslibrary.TinyDancer;
import com.example.konra.thespanishinquisition.MobileArrayAdapter;

public class LayoutActivity extends ListActivity {

    static final String[] MOBILE_OS =
            new String[] { "Android", "iOS", "WindowsMobile", "Blackberry", "Android"};

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setListAdapter(new MobileArrayAdapter(this, MOBILE_OS));

        TinyDancer.create()
                .redFlagPercentage(.1f) // set red indicator for 10%
                .startingGravity(Gravity.TOP)
                .startingXPosition(0)
                .startingYPosition(0)
                .show(this);

    }


}

File: WASMXamarin/WASMXamarin/WASMXamarin.Droid/LayoutActivity.cs
using System;

using Android.App;
using Android.OS;

namespace WASMXamarin.Droid
{
    [Activity(Label = "LayoutActivity")]
    public class LayoutActivity : ListActivity
    {
        static String[] MOBILE_OS = new String[] { "Android", "iOS", "WindowsMobile", "Blackberry", "Android"};
        protected override void OnCreate(Bundle savedInstanceState)
        {
            base.OnCreate(savedInstanceState);

            ListAdapter = new MobileArrayAdapter(this, MOBILE_OS);
        }
    }
}